apps/web/src/hooks/useExperimentalFeatureEnabled.ts

```diff
@@ -0,0 +1,17 @@
+import { EXPERIMENTAL_FEATURES } from 'config/experminetalFeatures'
+import Cookies from 'js-cookie'
+import { useEffect, useState } from 'react'
+
+const useExperimentalFeatureEnabled = (featureFlag: EXPERIMENTAL_FEATURES) => {
+  const [featureEnabled, setFeatureEnabled] = useState<boolean | null>(null)
+  const isCookieInFeatureFlags = () => Cookies.get(`ctx-${featureFlag}`)?.toString() === 'true'
```

Comment:
move this function outside of the component as it's pure utility function and has nothing to do with the state

Reply:
and single source of truth. Use`getCookieKey` from `apps/web/src/config/experminetalFeatures.ts`